### PR TITLE
AdminUI: Redesign of dialogs

### PR DESCRIPTION
**Pull request by stamenione**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

# Description

image

image

image

### CODE REVIEW

**Review by tnotheis — COMMENTED** [review]
I don't see a screenshot or any code changes for the login dialog. Has this already been correct or did you forget it?

**Comment by stamenione** [comment]
> I don't see a screenshot or any code changes for the login dialog. Has it already been correct or did you forget it?

It was already implement.. when the project was started. But i can add the screenshot of it no prob.

**Review by Siolto — APPROVED** [review]

File: AdminUi/apps/admin_ui/lib/core/modals/assign_quota_dialog.dart [hunk 1]
```diff
@@ -82,12 +82,23 @@ class _AddQuotaDialogState extends State<_AddQuotaDialog> {
     return PopScope(
       canPop: !_saving,
       child: AlertDialog(
-        title: Text(context.l10n.addQuota),
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(8),
+        ),
```
Comment (jkoenig134) [inline AdminUi/apps/admin_ui/lib/core/modals/assign_quota_dialog.dart:87]: 1 line

File: AdminUi/apps/admin_ui/lib/core/modals/assign_quota_dialog.dart [hunk 2]
```diff
@@ -82,12 +82,23 @@ class _AddQuotaDialogState extends State<_AddQuotaDialog> {
...
           width: 500,
           child: Column(
             mainAxisSize: MainAxisSize.min,
+            crossAxisAlignment: CrossAxisAlignment.start,
             children: [
+              Padding(
+                padding: const EdgeInsets.all(8),
+                child: Text(
+                  '*${context.l10n.required}',
+                ),
```
Comment (jkoenig134) [inline AdminUi/apps/admin_ui/lib/core/modals/assign_quota_dialog.dart:99]: 1 line

**Review by jkoenig134 — CHANGES_REQUESTED** [review]

File: AdminUi/apps/admin_ui/lib/l10n/app_en.arb [hunk 3]
```diff
@@ -37,6 +38,8 @@
   "period": "Period",
   "quotas": "Quotas",
   "retry": "Retry",
+  "required": "required",
+  "create": "Create",
```
Comment (jkoenig134) [inline AdminUi/apps/admin_ui/lib/l10n/app_en.arb:42]: 2 times create?!

File: AdminUi/apps/admin_ui/lib/core/modals/confirmation_dialog.dart [hunk 4]
```diff
@@ -24,16 +24,22 @@ class _ConfirmationDialog extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return AlertDialog(
-      title: Text(title),
-      content: Text(message),
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(8),
+      ),
```
[the remark this answers is not in the document]
Comment (stamenione) [reply AdminUi/apps/admin_ui/lib/core/modals/confirmation_dialog.dart]: @jkoenig134

File: AdminUi/apps/admin_ui/lib/core/modals/confirmation_dialog.dart [hunk 5]
```diff
@@ -24,16 +24,20 @@ class _ConfirmationDialog extends StatelessWidget {
...
       actions: [
         OutlinedButton(
           onPressed: () => Navigator.of(context).pop(false),
           child: Text(context.l10n.cancel),
         ),
         FilledButton(
           onPressed: () => Navigator.of(context).pop(true),
-          child: Text(context.l10n.confirm),
+          child: const Text('Remove'),
```
Comment (jkoenig134) [inline AdminUi/apps/admin_ui/lib/core/modals/confirmation_dialog.dart:40]: not properly translated

Comment (jkoenig134) [reply AdminUi/apps/admin_ui/lib/core/modals/confirmation_dialog.dart:40]: also I think b/c this is a confirmation dialog this should be a parameter.

Comment (jkoenig134) [reply AdminUi/apps/admin_ui/lib/core/modals/confirmation_dialog.dart:40]: (like the message)

File: AdminUi/apps/admin_ui/lib/core/modals/settings_dialog.dart [hunk 6]
```diff
@@ -18,24 +18,29 @@ class _SettingsDialog extends StatelessWidget with WatchItMixin {
...
-              ButtonSegment(value: ThemeMode.dark, icon: const Icon(Icons.dark_mode), label: Text(context.l10n.dark)),
-            ],
-            selected: {themeMode},
-            onSelectionChanged: (selected) => GetIt.I<ThemeModeModel>().setThemeMode(selected.first),
-          ),
-        ],
+      shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(8)),
+      title: Text(context.l10n.settings, textAlign: TextAlign.center),
+      content: Padding(
+        padding: const EdgeInsets.only(bottom: 32),
```
Comment (jkoenig134) [inline AdminUi/apps/admin_ui/lib/core/modals/settings_dialog.dart:24]: you should use `AlertDialog#contentPadding` instead to remove on layer of widgets, same for all other dialogs, make sure to use `const EdgeInsets.only(left: 24, right: 24, top: 20, bottom: 32)` to not override all defaults and only change the bottom.

See https://api.flutter.dev/flutter/material/AlertDialog/contentPadding.html for full docs

Comment (jkoenig134) [reply AdminUi/apps/admin_ui/lib/core/modals/settings_dialog.dart:24]: again, ping me if done

Comment (stamenione) [reply AdminUi/apps/admin_ui/lib/core/modals/settings_dialog.dart]: @jkoenig134

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**